rosdep/python.yaml

```
@@ -7552,6 +7552,15 @@ python3-openhsi-pip:
   ubuntu:
     pip:
       packages: [openhsi]
+python3-openpyxl:
+  alpine: [py3-openpyxl]
+  arch: [python-openpyxl]
+  debian: [python3-openpyxl]
+  fedora: [python3-openpyxl]
+  gentoo: [dev-python/openpyxl]
+  opensuse: [python3-openpyxl]
+  rhel: ['python%{python3_pkgversion}-openpyxl']
```

Yeah, this is a vestige which we don't need anymore:

```suggestion
  rhel: [python3-openpyxl]
```